ccp-openshift-scan/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.centos.org/centos/centos:latest
+
+RUN yum -y update && yum clean all
```

Comment:
It requires the RPM tracking to be available before we can start using this container. We can remove the step once RPM tracking is in action.

Reply:
But for now, we are building images on old infra which has RPM tracking available. Besides, I strongly feel that we should have RPM tracking in its basic form implemented sooner rather than later. However, in worst case, if we stick with keeping this line as-is, I'd prefer we create a card on OSIO to remove it at some point in time.

Reply:
`yum update -y` command makes the container image self-sufficient and not-relying on other services to update the contents. If other services are already updating the contents, this step will yield no data, but consume few seconds to return like `no updates available`. If we remove this line, and rely on other services to update the contents, I think we are putting a hard dependency on other services.

The tradeoff of removing a redundant step which costs few seconds to that putting a hard dependency on other service and its availability. I think we should choose here the redundant approach and let few seconds be spared, this we are sure about updated contents.

WDYT ?

Reply:
Just that in this case dependency is on our own service. :smiley:

Reply:
I would also vote as `yum -y update && yum clean all ` is redundant here.

Reply:
While we implement repo tracking in new arch, we're anyway using old arch which has repo tracking working just fine. So not sure what we achieve by keeping this line repo tracking is implemented in new arch. Unless we've decided to not do repo tracking in new arch for some time. Thoughts @bamachrn?

Reply:
I would say, if it becomes too complex we can skip repo tracking 1 sprint max, but we should implement things in a way to avoid re-work as far as possible.